manim/mobject/types/point_cloud_mobject.py

```diff
@@ -207,6 +207,18 @@ def __init__(self, *pmobs, **kwargs):
 
 
 class PointCloudDot(Mobject1D):
+    """A disc made of a cloud of Dots
+    Example
+    --------
+    .. manim:: PointCloudDotExample
+        :save_last_frame:
+
+        class PointCloudDotExample(Scene):
+            def construct(self):
+                dots = PointCloudDot(color=RED)
+                self.add(dots)
```

Comment:
I think a better example would be one that shows off more of the parameters
```suggestion
        class PointCloudDotExample(Scene):
            def construct(self):
                cloud_1 = PointCloudDot(color=RED)
                cloud_2 = PointCloudDot(stroke_width=4, radius=1)
                cloud_3 = PointCloudDot(density=15)
        
                group = Group(cloud_1, cloud_2, cloud_3).arrange()
                self.add(group)
```

Reply:
Can we keep them as two separate examples then. One for a quick generation and this for a more personalized generation with multiple `PointCloudDot`

Reply:
That wouldn't be a very effiecient use of space, especially since the original example is (no parameters) is covered in the example with more mobjects.

Reply:
Well the user can see the ways he/she can create this so that’s why I told we can have two examples

Reply:
I am also tending to the approach of using rendering time+space efficiently, and this example as @hydrobeam suggested is not too complex, so I am +1 for keeping it tight.